maven-compat/pom.xml

```diff
@@ -93,6 +93,11 @@ under the License.
       <artifactId>wagon-provider-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+    </dependency>
```

Comment:
I think it should be possible to inline the references to commons-io so that we actually trim down the list of dependencies and not add a new one...